wiki/Gameplay/Kiai_time/en.md

```diff
@@ -1,24 +1,33 @@
 ---
-stub: true
 tags:
-  - timing section
   - kiai mode
+  - kiai section
 ---
 
 # Kiai time
 
 *For regulations surrounding kiai time, see: [Ranking Criteria](/wiki/Ranking_Criteria)*
```

Comment:
that directory still hasn't been renamed <.<

```suggestion
*For regulations surrounding kiai time, see: [Ranking criteria](/wiki/Ranking_Criteria)*
```

Reply:
omg